src/Compilers/CSharp/Test/Symbol/Symbols/TypeTests.cs

```diff
@@ -30,14 +31,29 @@ public void Bug18280()
 
             string code = "class C {  int " + brackets + @" x; }";
 
-            var compilation = CreateCompilation(code);
+            var compilation = CreateCompilation(code, options: WithNonNullTypesTrue());
             var c = compilation.GlobalNamespace.GetTypeMembers("C")[0];
             var x = c.GetMembers("x").Single() as FieldSymbol;
-            var arr = x.Type;
+            var type = x.Type;
+            type.GetHashCode();
+            foreachArray(x => Assert.Equal(NullableAnnotation.NotAnnotated, x.ElementTypeWithAnnotations.NullableAnnotation));
+            type = type.SetObliviousNullabilityForReferenceTypes();
+            foreachArray(x => Assert.Equal(NullableAnnotation.Oblivious, x.ElementTypeWithAnnotations.NullableAnnotation));
```

Comment:
Minor point: It might be simpler to pass in the expected `NullableAnnotation` to the local function rather than an `Action`.